frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/general-tab.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import {
+  iGetRelevantTemplateSelectors,
+  iGetVmSettings,
+} from '../../selectors/immutable/vm-settings';
+import { VMSettingsField, VMWizardProps } from '../../types';
+import { FormFieldReviewMemoRow } from '../../form/form-field-review-row';
+import { iGetCommonData, iGetLoadedCommonData } from '../../selectors/immutable/selectors';
+import { getFieldValue } from '../../selectors/vm-settings';
+import { getField, getFlavorValue } from './utils';
+import { VMSettings } from '../../redux/initial-state/types';
+import { getOS } from '../../selectors/combined';
+
+import './review-tab.scss';
+
+const GeneralReviewConnected: React.FC<GeneralReviewConnectedProps> = (props) => {
+  const { iVMSettings, iUserTemplates, iCommonTemplates, openshiftFlag, relevantOptions, className } = props;
+
+  const flavorValue = getFlavorValue({
+    iVMSettings,
+    iUserTemplates,
+    iCommonTemplates,
+    relevantOptions,
+  });
+
+  const osName =
+    getOS({
+      osID: getFieldValue(iVMSettings.toJS() as VMSettings, VMSettingsField.OPERATING_SYSTEM),
+      iUserTemplates,
+      openshiftFlag,
+      iCommonTemplates,
+    })?.osName || '';
+
+  return (
+    <dl className={classNames('kubevirt-create-vm-modal__review-tab__data-list', className)}>
+      <FormFieldReviewMemoRow field={getField(VMSettingsField.NAME, iVMSettings)} />
+
+      <FormFieldReviewMemoRow field={getField(VMSettingsField.DESCRIPTION, iVMSettings)} />
```

Comment:
Should we be hiding description row completelyt when the description is missing? Wouldn't it make more sense to show it but empty with no value? @matthewcarleton @pcbailey

Reply:
I agree with showing it. Do you suggest leaving it blank or using the dashes approach we've used previously?

Reply:
ya if there's no description I'd say we keep it but have text like "empty" or "no description" those dashes don't work great for accessibility IMO

Reply:
Done!